server/data/strapiApiClient.ts

```diff
@@ -52,11 +50,11 @@ export default class StrapiApiClient {
     }
 
     return this.restClient
-      .get<ListResponse<Product>>({
+      .get<ListResponse<Strapi.Product>>({
```

Comment:
This reflects the general pattern: 

we receive a `ListResponse` of Strapi `Products` and then return an array of model `Products